clouddriver-openstack/src/test/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/view/OpenstackSecurityGroupProviderSpec.groovy

```diff
@@ -52,10 +54,8 @@ class OpenstackSecurityGroupProviderSpec extends Specification {
   Set<OpenstackSecurityGroup> allSecurityGroups = account1East.plus(account1West).plus(account2East).plus(account2West)
 
   def setup() {
-    accountCredentialsProvider = Mock()
-
-    // TODO Shouldn't need the spy once the data is coming from the caching layer
-    provider = Spy(OpenstackSecurityGroupProvider, constructorArgs: [accountCredentialsProvider])
+    provider = new OpenstackSecurityGroupProvider(cache, mapper)
+    cache.mergeAll(Keys.Namespace.SECURITY_GROUPS.ns, getAllCacheData())
   }
 
```

Comment:
What sort of exception test cases do you want? I am neither throwing or handling any checked exceptions in this class.